File: app/src/main/java/cenergy/central/com/pwb_store/adapter/viewholder/CompareHeaderDetailViewHolder.java
package cenergy.central.com.pwb_store.adapter.viewholder;

import androidx.recyclerview.widget.RecyclerView;
import android.view.View;

import cenergy.central.com.pwb_store.R;
import cenergy.central.com.pwb_store.model.CompareDetail;
import cenergy.central.com.pwb_store.view.PowerBuyTextView;

/**
 * Created by napabhat on 7/31/2017 AD.
 */

public class CompareHeaderDetailViewHolder extends RecyclerView.ViewHolder {

    PowerBuyTextView header;

    public CompareHeaderDetailViewHolder(View itemView) {
        super(itemView);
        header = itemView.findViewById(R.id.textView);
    }

    public void setViewHolder(CompareDetail compareDetail){
        header.setText(compareDetail.getName());
    }
}

File: app/src/main/java/cenergy/central/com/pwb_store/adapter/viewholder/CompareHeaderDetailViewHolder.kt
package cenergy.central.com.pwb_store.adapter.viewholder

import android.view.View
import androidx.recyclerview.widget.RecyclerView
import cenergy.central.com.pwb_store.R
import cenergy.central.com.pwb_store.adapter.CompareDetailAdapter
import kotlinx.android.synthetic.main.list_item_text_header_compare_detail.view.*

class CompareHeaderDetailViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    private val tvLabel = itemView.labelTextView
    fun bind(item: CompareDetailAdapter.CompareTitleItem) {
        if (item.title == "Short Description"){
            tvLabel.text = itemView.context.getString(R.string.product_overview)
        } else {
            tvLabel.text = item.title
        }
    }
}
